lib/rules/sotd/pp.js

```diff
@@ -103,6 +122,12 @@ exports.check = function (sr, done) {
     ,     pp2017 = "https://www.w3.org/Consortium/Patent-Policy-20170801/"
     ,     patent = (sr.config.patentPolicy === "pp2020") ? pp2020 : (sr.config.patentPolicy === "pp2004") ? pp2017: sr.config.patentPolicy
     ;
+    var groupName = await fetchGroupName(sr.getDelivererIDs());
+    sr.isIGDeliverable = groupName.endsWith('Interest Group');
+
+    // fetchGroupName(sr.getDelivererIDs()).then(function (groupName) {
```

Comment:
Delete?

Reply:
Same reason I leave comment here, to discuss whether use `await` or Promise `then`. Now I tend to use `await` to make the code easy to read.